deploy-board/deploy_board/templates/configs/env_config.tmpl

```diff
@@ -247,6 +247,19 @@
                         </select>
                     </div>
                 </div>
+                <div class="form-group">
+                    <label for="syspriority" class="deployToolTip control-label col-xs-2"
+                        data-toggle="tooltip"
+                        title="System Priority for Sidecars">
+                        Sidecar System Priority
+                    </label>
+
+                    <div class="col-xs-4">
```

Comment:
Can we just put the value in label?

Reply:
we can reserve the option to make the field writable in the future